### DISCUSSION TITLE
GDPR (checkbox) Field not recognised

### DISCUSSION THREAD

**Discussion by fredericalpers** [onOffice-Web-Org/oo-wp-plugin#454] [_Discarded pitches]
### Current State

The GDPR (checkbox) field is not recognised when initially saved in WP Backend.

If you add the GDPR (checkbox) field in the form, it is not displayed in the frontend until another change is made in the form. For instance, the order must be changed again or another field must be added.

### Desired State

The GDPR (checkbox) field should be recognised on first save when added to the form in WP Backend.

![gdpr-checkbox-added-backend](https://user-images.githubusercontent.com/88546396/219674199-fcf052ba-ffea-4715-a52a-b632e1c258e5.PNG)

![gdpr-no-checkbox-frontend](https://user-images.githubusercontent.com/88546396/219674178-0ab62623-e7ed-4c29-96ea-7f8baa3c2769.PNG)

![gdpr-checkbox-moved-backend](https://user-images.githubusercontent.com/88546396/219674195-8ec171a5-a395-49c2-b441-03116f2419ac.PNG)

![gdpr-checkbox-frontend](https://user-images.githubusercontent.com/88546396/219674187-264195fd-0c2d-42d8-bea7-9488740560e4.PNG)

**Comment by dai-eastgate** [comment]
@fredericalpers I fixed this bug, you can watch the video and give me feedback. 

https://user-images.githubusercontent.com/106214469/224212516-c263a06c-03e5-479c-b57e-e436194903e6.mp4

**Reply by fredericalpers** [reply]
Seems good! Let me consult with our devs on how we will proceed from here.

**Comment by dai-eastgate** [comment]
fixed at https://github.com/onOffice-Web-Org/oo-wp-plugin/pull/460 and code merged to master
